doc/api/next_api_changes/behavior/23573-AL.rst

```diff
@@ -0,0 +1,5 @@
+All Axes have a ``get_subplotspec`` method now, which returns None for Axes not positioned via a gridspec
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Previously, this method was only present for Axes positioned via a gridspec.
+Following this change, checking ``hasattr(ax, "get_gridspec")`` should now be
+replaced by ``ax.get_gridspec() is not None``.
```

Comment:
Not sure if it needs mentioning but if you need compatibility with the old and new behavior you can replace `hasattr(ax, "get_gridspec")` by `getattr(ax, "get_gridspec", None) is not None`.

Reply:
sure.